cli/lsp/resolver.rs

```diff
@@ -484,8 +431,19 @@ fn create_graph_resolver(
     bare_node_builtins_enabled: config_file
       .map(|cf| cf.has_unstable("bare-node-builtins"))
       .unwrap_or(false),
-    // not used in the LSP as the LspGraphResolver handles this
-    sloppy_imports_resolver: None,
+    sloppy_imports_resolver: unstable_sloppy_imports.then(|| {
+      SloppyImportsResolver::from_stat_sync_fn(Box::new(|path| {
+        path.metadata().ok().and_then(|m| {
+          if m.is_file() {
+            Some(SloppyImportsFsEntry::File)
+          } else if m.is_dir() {
+            Some(SloppyImportsFsEntry::Dir)
+          } else {
+            None
+          }
+        })
+      }))
```

Comment:
Actually doesn't this callback just act the same as `SloppyImportsResolver::from_fs(Arc::new(deno_fs::RealFs))`?

Reply:
It used to not because it was using the open documents and I changed that. We could align them now.